webapp/channels/src/components/searchable_channel_list.tsx

```diff
@@ -162,8 +161,8 @@ export class SearchableChannelList extends React.PureComponent<Props, State> {
         ) : null;
 
         const channelPurposeContainerAriaLabel = localizeAndFormatMessage(
```

Comment:
Best would be to remove `localizeAndFormatMessage` altogether. Was it not possible here?

Reply:
It might be possible, but it's not easy. It's on my list of things to tackle next once `t` is gone though since it's also preventing us from switching to the FormatJS tooling